src/AstriaBridgeableERC20Manager.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: MIT or Apache-2.0
+pragma solidity ^0.8.21;
+
+import {AstriaBridgeableERC20} from "./AstriaBridgeableERC20.sol";
+import {Ownable} from "lib/openzeppelin-contracts/contracts/access/Ownable.sol";
+
+contract AstriaBridgeableERC20Manager is Ownable {
+
+  struct BridgeableErc20Asset {
+    uint256 startHeight;
+    bytes32 astriaAssetDenominationHash;
+    address erc20Contract;
+  }
+
+  mapping(string => BridgeableErc20Asset) public bridgeableErc20Assets;
+
+  modifier onlyErc20Bridge(address _bridgeableERC20Address) {
```

Comment:
```suggestion
  modifier onlyErc20BridgeCaller(address _bridgeableERC20Address) {
```

the naming makes it sound like only a bridge contract can call, whereas it's actually only the internal bridge "caller" address that can call